tools/crawl_summary/crawl_report.py

```diff
@@ -0,0 +1,186 @@
+from collections import Counter
+import datetime
+import os
+import inspect
+import json
+
+STR_DATETIME_FORMAT = "%d_%m_%y_%H_%M_%S"
+
+
+class CrawlReport:
+    def __init__(self) -> None:
+        path_to_dir = os.sep.join(inspect.getfile(self.__class__).split(os.sep)[:-1])
+        self.default_save_dir = os.path.join(path_to_dir, "crawl_data")
+        self.data = {"lang": {}}
+        self.crawl_date = datetime.datetime.now()
+
+    def get_days_from_crawl_date(self):
+        return (datetime.datetime.now() - self.crawl_date).days
+
+    def __convert_str_to_dt(self, str_datetime):
+        dt_parsed = datetime.datetime.strptime(str_datetime, STR_DATETIME_FORMAT)
+        return dt_parsed
+
+    def __convert_dt_to_str(self, datetime):
```

Comment:
I decided to make it a private method for the class, but maybe I will make it a class method to reflect this. This is just to ensure it's converted in the expected format for the file output.

Let me know if you agree with it otherwise I can make it part of the methods that save and load, it's just in my mind I see it as a pair the dt_to_str and str_to_dt